pkg/apiclient/apiclient.go

```diff
@@ -402,7 +402,8 @@ func (c *client) refreshAuthToken(localCfg *localconfig.LocalConfig, ctxName, co
 	if err != nil {
 		return err
 	}
-	if claims.Valid() == nil {
+	validator := jwt.NewValidator(jwt.WithLeeway(5 * time.Second))
+	if validator.Validate(claims) == nil {
```

Comment:
I'm not sure we should set a 5s leeway here. I don't think there was any leeway in v4 (https://github.com/golang-jwt/jwt/compare/v4.5.1...v5.2.1#diff-9803198a7ae31435cd378cab1ca22b1c0bb28c58c9d9b308d246ac96982bdc95L250), and I think the `5 * time.Second` example in the migration guide is just an example. https://github.com/golang-jwt/jwt/blob/main/MIGRATION_GUIDE.md